otherlibs/win32unix/envir.c

```diff
@@ -21,23 +21,26 @@
 #include <caml/osdeps.h>
 
 #include <Windows.h>
+#include <Stdlib.h>
 
 CAMLprim value unix_environment(value unit)
 {
-  CAMLparam0();
-  CAMLlocal2(v, result);
-  wchar_t * envp, * p;
-  int size, i;
+  CAMLparam0 ();
+  CAMLlocal2 (v, result);
```

Comment:
Hi!

Are the spaces after the function names introduced by design?

It's on line 28, 29, 36.

Reply:
Thanks for the comment; and not, not really. Will fix it and give AppVeyor an excuse to wake up again.